CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 
 * **New Resource:** sumologic_hierarchy (GH-260)
+* **New Resource:** sumologic_cse_network_block
```

Comment:
Can you also add PR number like we have for `sumologic_hierarchy`?

Reply:
done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/271/commits/4c00657834f23744c79da15a651cf03f35b3f5dc